app/states/testState.js

```diff
@@ -1,43 +1,55 @@
 import StackQuest from '../main'
 
-let cursors
-  , backgroundText
+let map
+  , layer
+  , cursors
   , fixedText
 
 const testState = {
   preload() {
-
+    StackQuest.load.tilemap('testmap', 'maps/testmap.json', null, Phaser.Tilemap.TILED_JSON)
```

Comment:
StackQuest can be replaced by this. Referring to StackQuest in the different *State.js files and referring to the *State.js files in StackQuest gives the 'appearance' of circular dependencies.